samples/subsys/display/lvgl/README.rst

```diff
@@ -16,6 +16,14 @@ additional widgets may be displayed and additional interactions enabled:
       If your board has a touch panel controller
       (:dtcompatible:`zephyr,lvgl-pointer-input`), a button widget is displayed
       in the center of the screen. Otherwise a label widget is displayed.
+
+      The demo also features drawing functionality, which can to be enabled
+      via menuconfig (``SAMPLE_LVGL_TOUCH_DRAWING``). When enabled, received touch
```

Comment:
@kartben I don't know it by heart but I think there is a way to link to the kconfig symbol `CONFIG_` instead?

Reply:
There is (``:kconfig:option:`...` ``, but it doesn't make sense in the context of a sample-specifc Kconfig since the option does not appear in the documentation hence does not make sense linking to.
Good question though, and I have to admit I too did a double-take initially and almost suggested to add a link myself :)